nowledge-mem-bub-plugin/src/nowledge_mem_bub/client.py

```
@@ -103,17 +101,15 @@ async def _exec(
                 stderr=asyncio.subprocess.PIPE,
                 env=self._build_env(),
             )
-            stdout, stderr = await asyncio.wait_for(
-                proc.communicate(), timeout=timeout
-            )
+            stdout, stderr = await asyncio.wait_for(proc.communicate(), timeout=timeout)
         except asyncio.TimeoutError:
             proc.kill()  # type: ignore[union-attr]
             raise NmemError(f"nmem timed out after {timeout}s")
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)